### ISSUE TITLE
Can not affect "Alt+Esc" as dropdown key binding

### ISSUE THREAD

**Issue by Arnaudv6** [lxqt/qterminal#281]
Hello there and thanks for this nice and lightweight soft !
I wanted to affect my usual dropdown key binding to qterminal,
but it doesn't seem to accept any binding with Esc key in it, combined with others or not.

**Comment by stefonarch** [comment]
Esc should and can most probably not be used as part of any shortcut.

**Comment by tsujan** [comment]
> Esc should and can most probably not be used as part of any shortcut.

I agree, but what about `Alt+Esc`, `Ctrl+Esc`,....  "Preferences → Shortcuts" allows them. I think this is a valid feature request.

**Comment by tsujan** [comment]
Moreover, "Preferences → Dropdown → Shortcut" has always been weird to me. It should be a real shortcut entry, instead of a simple text entry, IMO.

**Comment by stefonarch** [comment]
True both, I thought about ESC as first item, usually windows exit immediately when trying to add it.

**Comment by tsujan** [comment]
Might work on both later.

**Comment by tsujan** [comment]
@stefonarch 
I think you were right. Escape can't work as a shortcut because it's used by the terminal itself, as a control character.

**Comment by stefonarch** [comment]
it's a minor problem. In the meantime I realized that I use ctrl+Esc for qps ;)
Change the textedit in the setting is more important probably.

**Comment by tsujan** [comment]
> I realized that I use ctrl+Esc for qps 

It's different. Here we deal with a terminal emulator.

**Comment by tsujan** [comment]
> Change the textedit in the setting is more important probably.

It's a piece of cake. But, while doing it, I encountered two bugs ;) I mean https://github.com/lxqt/qterminal/issues/905 and https://github.com/lxqt/qterminal/issues/904

**Comment by stefonarch** [comment]
Closing one we open two new ones atm ;)

**Comment by tsujan** [comment]
The two bugs are fixed. Will add the shortcut entry tomorrow; it needs a small customization to be consistent with other entries.

**Comment by Arnaudv6** [comment]
Thanks for support, @tsujan.
For the record, I've personally lost interest in this (and wayland won't allow for desktop-wide binding registration AFAICT)

**Comment by tsujan** [comment]
> and wayland won't allow for...

We should disable the drop-down mode under Wayland for now. Even if we fix its crash under Wayland, there's no way of positioning windows there.
